plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/TestRemarksReportingOraclePoolConnectorTest.java

```diff
@@ -13,24 +13,20 @@
  */
 package io.trino.plugin.oracle;
 
-import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
 import io.airlift.testing.Closeables;
 import io.trino.testing.QueryRunner;
 import io.trino.testing.sql.SqlExecutor;
+import io.trino.tpch.TpchTable;
 import org.testng.annotations.AfterClass;
 
 import java.io.IOException;
 
 import static io.trino.plugin.oracle.TestingOracleServer.TEST_PASS;
 import static io.trino.plugin.oracle.TestingOracleServer.TEST_USER;
-import static io.trino.tpch.TpchTable.CUSTOMER;
-import static io.trino.tpch.TpchTable.NATION;
-import static io.trino.tpch.TpchTable.ORDERS;
-import static io.trino.tpch.TpchTable.REGION;
 
-public class TestOraclePoolIntegrationSmokeTest
-        extends BaseOracleIntegrationSmokeTest
+public class TestRemarksReportingOraclePoolConnectorTest
```

Comment:
Possibly we want to have a real "smoke" base class for this one. 
Maybe add `//TODO` with an issue. Not sure how we want to proceed with that. So we don't get where we started. And have  two arbitrary split test classes. cc: @findepi